internal/pkg/cli/deploy/workload.go

```diff
@@ -56,6 +63,12 @@ const (
 	labelForContainerName = "com.aws.copilot.image.container.name"
 )
 
+const (
+	// pollInterval is the time Interval to wait between checking whether the output buffers are done.
+	pollInterval = 60 * time.Millisecond
```

Comment:
nit: since this is a big package, these should probably be named more specific to what their usage is - `pollInterval` sounds fairly generic